src/ApiController.java

```diff
@@ -0,0 +1,63 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+public class ApiController {
+
+    public String runJsonQuery(String query){
+        String result = null;
+        try {
+            // Replace the URL with the actual GraphQL endpoint
+            URL apiUrl = new URL("http://localhost:8081/graphql");
+
+            // Replace this with your custom GraphQL query
+            String jsonRequest = "{\"query\": \"" + escapeDoubleQuotes(query) + "\",\"operationName\": \"MyQuery\"}";
+
+
+            // Open a connection to the URL
+            HttpURLConnection connection = (HttpURLConnection) apiUrl.openConnection();
+
+            // Set the request method to POST
+            connection.setRequestMethod("POST");
```

Comment:
moet dit niet een GET request zijn? POST is om dingen toe te voegen of te vervangen.

Reply:
nee het werkt via een POST waarom moet je aan berne vragen